CHANGELOG.md

```diff
@@ -3,6 +3,28 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[3.3](https://github.com/nf-core/rnaseq/releases/tag/3.3)] - 2021-07-28
+
+### Enhancements & fixes
+
+* Updated pipeline template to [nf-core/tools 2.1](https://github.com/nf-core/tools/releases/tag/2.1)
+* [[#668](https://github.com/nf-core/rnaseq/issues/668)] - Salmon quant with UMI-tools does not work
+* [[#674](https://github.com/nf-core/rnaseq/issues/674)] - Launch pipeline regex fails
+
+### Software dependencies
+
+Note, since the pipeline is now using Nextflow DSL2, each process will be run with its own [Biocontainer](https://biocontainers.pro/#/registry). This means that on occasion it is entirely possible for the pipeline to be using different versions of the same tool. However, the overall software dependency changes compared to the last release have been listed below for reference.
+
+| Dependency  | Old version | New version |
+|-------------|-------------|-------------|
+| `samtools`  | 1.10        | 1.12        |
```

Comment:
Maybe we could think of just refer here to the `modules.json` in the future 🤔

Reply:
Good point 👍🏽 Problem is that listing SHA hashes here probably wouldn't be suited for a CHANGLOG! 🤯 But maybe we could list the container address in `modules.json` and try and look for a `diff` between releases to get an automated listing of what has changed instead of getting this info manually!! Would you mind dumping this in an issue on tools? 🚀

Reply:
Sure!

Reply:
Here is the issue:

https://github.com/nf-core/modules/issues/635